.babelrc

```diff
@@ -1,3 +1,15 @@
 {
-  "presets" : ["es2015", "react"]
```

Comment:
lol i messed up on reviewing pls see other comments

Reply:
wait i updated babel so now es2015 is deprecated